src/super_gradients/recipes/dataset_params/imagenet_resnet50_dataset_params.yaml

```diff
@@ -0,0 +1,32 @@
+defaults:
+  - imagenet_dataset_params
+
+resize_size: 236
+random_erase_prob: 0
+random_erase_value: random
+train_interpolation: random
+rand_augment_config_string: rand-m7-mstd0.5
+cutmix: True
+cutmix_params:
+  mixup_alpha: 0.2
+  cutmix_alpha: 1.0
+  label_smoothing: 0.1
+
+
+# TODO: REMOVE ABOVE, HERE FOR COMPATIBILITY UNTIL WE REMOVE DATASET_INTERFACE
+#defaults:
+#  - imagenet_dataset_params
+
+train_transform_args:
```

Comment:
This is definitely not what we are going for...
We wanted the transforms to be visible for the user and this does exactly the same as dataset interface.

Its ok that we have some parenting args to help the user easily adapt transforms (such as input size), but definitely not this.

Reply:
We talked about it and the problem is that we cannot modify an element in a list, so if we want to modify any transform we have to redefine the whole list of transforms.

So either we always repeat all the transforms (redundant but explicit)
Or we create this "train_transform_args", that acts like an interface to not have to repeat all the transforms (shorter but implicit)

Reply:
I changed according to our discussion: redefine all the transforms when one is changed, in order to have explicit and easy to understand recipes